packages/cli/src/services/project.service.ee.ts

```diff
@@ -537,4 +609,13 @@ export class ProjectService {
 	async getProjectCounts(): Promise<Record<ProjectType, number>> {
 		return await this.projectRepository.getProjectCounts();
 	}
+
+	/** Type guard for assignable project role slugs */
+	private isAssignableProjectRoleSlug(slug: string): slug is AssignableProjectRole {
```

Comment:
This typeguard is not correct, since we also have custom roles, which can be an arbitrary string. We could use the zod schema to implement the typeguard here.